modules/configuring/con-dynamic-plugins-dependencies.adoc

```diff
@@ -0,0 +1,74 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="dynamic-plugins-dependencies_{context}"]
+= Dynamic plugins dependency management
+Dynamic plugins configured for the {backstage} custom resource (CR) may require certain Kubernetes resources to be configured to make the plugin work. These resources are referred to as 'plugin dependencies'. In {product}({product-very-short}), you can automatically create these resources when the {backstage} CR is applied to the cluster.
+
+== Profile Configuration
```

Comment:
I would omit mentioning Profile term here as we do not describe it in downstream doc.
Let's say something like this:
== Cluster level plugin dependencies configuration 
Same as default configuration (maybe link here), dynamic plugins dependency implies Kubernetes manifests for dependency resource to be created as a part of operator (cluster level) configuration. by including....